src/wp-includes/fonts.php

```diff
@@ -188,3 +188,13 @@ function _wp_before_delete_font_face( $post_id, $post ) {
 		wp_delete_file( $font_dir . '/' . $font_file );
 	}
 }
+
+/**
+ * Register the default font collections.
+ *
+ * @access private
+ * @since 6.5.0
+ */
+function _register_font_collections() {
```

Comment:
Nit: every other function in the file has prefix `wp_` or `_wp_`. Shouldn't we conform? Also, would it be more descriptive to include the adjective _default_?

```
_wp_register_default_font_collections()
```

edit: actually, that's exactly what the doc says :)